app/views/faqs/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-sm-3 side-images">
+    <%= image_tag('boats.jpg', alt: 'small wooden boxes with tiny boats and waves inside that are controlled by a crank', class: 'content-container side-image') %>
+    <%= image_tag('bottles.jpg', alt: 'tiny rows of bottle stoppers with glass vessels on top with natural items inside', class: 'content-container side-image')%>
```

Comment:
same comment as last time. not sure what this is describing. it doesn't give me a visual